SphereSurfaceChargeDensity/MathOpsModule.py

```diff
@@ -0,0 +1,23 @@
+""" This Module is to define the general mathematical operations to be used by the other modules """
+
+def div(numerator, denominator):
+    """
+    Returns the result of dividing two numbers
+    Parameters:
+    numerator (float), denominator (float): input numbers
```

Comment:
if float is not actually required or checked do not use it in the docstring

Reply:
Noted.